api/constants/constants.go

```diff
@@ -239,3 +239,80 @@ const (
 	// session recording fails.
 	SessionRecordingModeBestEffort = SessionRecordingMode("best_effort")
 )
+
+// Constants for Traits
+const (
+	// TraitInternalPrefix is the role variable prefix that indicates it's for
+	// local accounts.
+	TraitInternalPrefix = "internal"
+
+	// TraitExternalPrefix is the role variable prefix that indicates the data comes from an external identity provider.
+	TraitExternalPrefix = "external"
+
+	// TraitLogins is the name of the role variable used to store
+	// allowed logins.
+	TraitLogins = "logins"
+
+	// TraitWindowsLogins is the name of the role variable used
+	// to store allowed Windows logins.
+	TraitWindowsLogins = "windows_logins"
+
+	// TraitKubeGroups is the name the role variable used to store
+	// allowed kubernetes groups
+	TraitKubeGroups = "kubernetes_groups"
+
+	// TraitKubeUsers is the name the role variable used to store
+	// allowed kubernetes users
+	TraitKubeUsers = "kubernetes_users"
+
+	// TraitDBNames is the name of the role variable used to store
+	// allowed database names.
+	TraitDBNames = "db_names"
+
+	// TraitDBUsers is the name of the role variable used to store
+	// allowed database users.
+	TraitDBUsers = "db_users"
+
+	// TraitAWSRoleARNs is the name of the role variable used to store
+	// allowed AWS role ARNs.
+	TraitAWSRoleARNs = "aws_role_arns"
+
+	// TraitTeams is the name of the role variable use to store team
+	// membership information
+	TraitTeams = "github_teams"
+
+	// TraitJWT is the name of the trait containing JWT header for app access.
+	TraitJWT = "jwt"
+
+	// TraitInternalLoginsVariable is the variable used to store allowed
+	// logins for local accounts.
+	TraitInternalLoginsVariable = "{{internal.logins}}"
```

Comment:
Do we need to move the variable traits? Are they useful without expansion? I would keep those under teleport/ if the answers are "no" or unclear.

Same for any Traits we don't explicitly need - adding surface is easier than removing, if we so desire.

Reply:
I reverted and only added the ones we use
Thank you :+1: